components/src/lib/hooks/useFocusTrap.ts

```
@@ -0,0 +1,77 @@
+          e.preventDefault();
+          firstElement.focus();
+        }
+      }
+    }
+  };
+
+  useEffect(() => {
+    const originalStyle = window.getComputedStyle(document.body).overflow;
+    document.body.style.overflow = 'hidden';
+
+    if (initialFocusRef && initialFocusRef.current) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)